### ISSUE TITLE
InstantiateMainScene() breaking change

### ISSUE THREAD

**Issue by Daniel4144** [atteneder/glTFast#500]
Version 5 introduces async scene instantiation by making the existing InstantiateMainScene() async, which could break existing projects. Maybe you want to consider adding a new InstantiateMainSceneAsync() instead?

**Comment by atteneder** [comment]
Thanks for your feedback. I'll give it a thought before releasing.

**Comment by atteneder** [comment]
Ok, so here's my stance:

Writing sync wrappers for async methods is an [anti-pattern](https://devblogs.microsoft.com/pfxteam/should-i-expose-synchronous-wrappers-for-asynchronous-methods/).

However, I acknowledge that forcing users to change their code immediately upon upgrading isn't ideal.

I'll cover the topic in the upgrade guide, add the sync wrapper methods, flagging them as `Obsolete` with links to said upgrade guide.

The sync methods will get removed in some future major release.

Let me know if that sounds good to you.

Relates to changes from #205

**Comment by Daniel4144** [comment]
Imo that's a good solution.

Btw this is not an urgent requirement for me. I noticed this and thought I share it here.
I noticed it when experimenting with Mixed Reality Toolkit (which uses glTFast for loading models of VR controllers) and I accidently imported the latest preview version of glTFast. It would be an easy fix in that case by just adding the await keyword, but editing code of Unity packages is not so easy and in case of several dependencies to different versions of glTFast problems would add up.
